source/channels/organize-conversations.rst

```diff
@@ -89,10 +72,17 @@ Select **Unreads** to filter your followed threads by only those with unread rep
 Tutorial video
 ---------------
 
-For additional details about working with the beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <https://www.youtube.com/watch?v=hnD0Zj-mIbs>`_. 
+For additional details about working with Collapsed Reply Threads, see our `tutorial video on YouTube <https://www.youtube.com/watch?v=hnD0Zj-mIbs>`_. 
```

Comment:
New vid link coming from @rmonga71

Reply:
Thanks Carrie. We should have it by EOD tomorrow (Tuesday)